opflexagent/test/test_endpoint_file_manager.py

```diff
@@ -1084,3 +1084,27 @@ def test_bad_mapping(self):
         # Get rid of the EPG, verify we don't get an exception
         mapping['endpoint_group_name'] = None
         self.manager.declare_endpoint(port_1, mapping)
+
+    def test_svi_port_bound(self):
+        # the SVI related info we expect to see
+        # on get_gbp_details
+        svi_info = {}
+        svi_info['svi'] = True
+        svi_info['svi_vlan'] = 1234
+        svi_info['endpoint_group_name'] = 'svi-net-id'
+        # knock off some extraneous stuff
+        svi_info['floating_ip'] = []
```

Comment:
Do we not support floating IPs or SNAT with SVI?

Reply:
No its not that, for the purposes of testing this change was keeping things more focussed on the relevant fields. But perhaps it gives the wrong idea ?

Reply:
Just seemed a bit odd that these were explicitly removed. I'd keep them in if it doesn't affect anything.

Reply:
Updated as this is not as relevant to the change.